build.gradle

```diff
@@ -10,13 +10,15 @@ buildscript {
 
     ext.versions = [
             'kotlin': '1.8.0',
-            'okhttp': '3.14.7',
-            'bouncycastle': '1.61',
+            'okhttp': '4.12.0',
+            'bouncycastle': '1.69',
```

Comment:
```suggestion
```

Reply:
there are two suggestions, then you removed bouncycastle, but it's used. Can you clarify?

Reply:
I've also changed it to 1.72 and `jdk18on` which doesn't have vulnerabilities

Reply:
![image](https://github.com/doordeck/doordeck-sdk-java/assets/6842786/79065190-c8ba-41d0-9cd7-c74f9106051a)

Reply:
looks like defined globally, and I think it was done on purpose. I can separate them by module